### PR TITLE
[Shimmer] Update docs

### PR DESCRIPTION
**Pull request by ayush547**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Updated Shimmer documentation.

### Verification

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: packages/experimental/Shimmer/src/Shimmer.types.ts [hunk 1]
```diff
@@ -89,20 +89,20 @@ export type ShimmerElementTypes = ShimmerCircleElement | ShimmerRectElement;
...
-   * @defaultValue 0
+   * @defaultValue 500 on win32, 0 on other platforms
    */
   delay?: number;
 
   /**
    * Specifies the time required to traverse the control in milliseconds
    *
-   * @defaultValue 2000
+   * @defaultValue 1000 on Android, 2000 on win32, 7000 on other platforms
```
Comment (Saadnajmi) [inline packages/experimental/Shimmer/src/Shimmer.types.ts:105]: Why are the defaults different across platforms? I thought these all basically had the same designs?

Comment (ayush547) [reply packages/experimental/Shimmer/src/Shimmer.types.ts:105]: For Android this was based on design guidelines.
While existing code for win32 and other platforms had the value 2000 and 7000 respectively.

File: packages/experimental/Shimmer/SPEC.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+## Background
+
+The `Shimmer` component is a temporary animation placeholder for when a service call takes time to return data and we don't want to block rendering the rest of the UI.
+
+Many a time, applications must wait for content to load or perform lengthy data processing operations. During these times, it is vital to show the user that the application isn't stalled. Progress indicators like `Shimmer` are used in these cases.
+
+## Requirements
+
+If using FURN's theming, the `Shimmer` requires use of the `ThemeProvider` from `@fluentui-react-native/theme` to work properly with themes. Please see [this page](../../../docs/pages/Guides/UpdateThemeProvider.md) for information on updating your `ThemeProvider` if using the version from `@uifabricshared/theming-react-native`.
```
Comment (ankraj12) [inline packages/experimental/Shimmer/SPEC.md:11]: We can consider adding the reference to current `ThemeReference` as well. Probably this - https://github.com/microsoft/fluentui-react-native/blob/main/docs/pages/Theming/Basics.md

**Review by ankraj12 — APPROVED** [review]

File: packages/experimental/Shimmer/SPEC.md (hunk 2 above)
Comment (ayush547) [reply packages/experimental/Shimmer/SPEC.md]: Modified

**Pull request merged**